NativeApp/src/Android/AndroidAppBase.cpp

```diff
@@ -118,22 +119,60 @@ void AndroidAppBase::HandleCmd(struct android_app* app, int32_t cmd)
     AndroidAppBase* eng = (AndroidAppBase*)app->userData;
     switch (cmd)
     {
-        case APP_CMD_SAVE_STATE:
-            break;
-
         case APP_CMD_INIT_WINDOW:
-            // The window is being shown, get it ready.
-            if (app->window != NULL)
+#if (DILIGENT_DEVLOPMENT)
```

Comment:
I actually think it makes sense to always log this - there is really no reason not to...

Reply:
hmm ok

Reply:
logging of `APP_CMD_INIT_WINDOW` and not `INIT DISPLAY - HAS SURFACE` right?

Reply:
eg

```
        case APP_CMD_INIT_WINDOW:
            LOG_INFO_MESSAGE("APP_CMD_INIT_WINDOW");
```

?

Reply:
logging of all events

Reply:
Yes

Reply:
hmm ok

Reply:
how is this?